AstSemantics.md

```diff
@@ -225,6 +226,9 @@ The details of index space for local variables and their types will be further c
 e.g. whether locals with type `i32` and `i64` must be contiguous and separate from
 others, etc.
 
+Finally, because local variables can be seen as virtual registers, `get_local` and
```

Comment:
This is a lot of low-level detail that is probably only interesting to engine implementors and compiler people, so I don't think that it is necessary here.